Junk4Dollars-app/ShowAuctions/AuctionListTableViewDataSource.swift

```diff
@@ -0,0 +1,21 @@
+import UIKit
+
+public class AuctionListTableViewDataSource: NSObject, UITableViewDataSource {
```

Comment:
Do you need `NSObject`?

Reply:
Yeah, it wouldn't work without it. Eric, we should talk about this though.